cargo/crate_universe_resolver/src/templates/crate.BUILD.template

```diff
@@ -0,0 +1,40 @@
+# buildifier: disable=load
+load(
+    "@rules_rust//rust:rust.bzl",
```

Comment:
What do you think about using the "more focused" rules from //rust:defs.bzl?

Reply:
Definitely - I'll make that change now, these didn't exist when I put together this branch :)